docs/background/object-storage.md

```diff
@@ -0,0 +1,63 @@
+
+## Object storage integration with OpenStack authentication
+
+In {{brand}}, the OpenStack `object‑store` service endpoints point not to native OpenStack Swift proxy servers, but to radosgw endpoints.
+This is to say that radosgw acts as a "drop-in" replacement for OpenStack Swift.
+
+As such, it is fully integrated with OpenStack's authentication facility, OpenStack Keystone.
+This means that once you have configured the [OpenStack CLI](../howto/getting-started/enable-openstack-cli.md) correctly, you are able to interact with the `object‑store` endpoint as with any other OpenStack service, using the Swift API.
+
+The S3 API, however, does not natively "know" about OpenStack authentication.
+The radosgw endpoints in {{brand}} *still* authenticate via Keystone, albeit taking a little detour:
+you must first [create AWS-style credentials](../howto/object-storage/s3/credentials.md) and configure your S3 client with them, which it can then use to make properly authenticated S3 API calls.
+
+
+## Multi-tenancy
+
+OpenStack Swift has the concept of multi-tenancy built-in.
+Thus, if you use the Swift API with credentials for an [OpenStack user](../howto/getting-started/enable-openstack-cli.md) that authenticates against a specific {{brand}} *region* and uses a specific *project*, it can only interact with objects stored in containers belonging to that project by default.
+
+In {{brand}}, you also get multi-tenancy for S3 objects, even though this concept is not intrinsic to S3 itself.
+Here, again, a small conceptual detour applies: when you [create AWS-style credentials](../howto/object-storage/s3/credentials.md), then those are linked to a specific OpenStack user, region, and project.
+When you subsequently use those credentials for S3 interactions, any buckets and objects thus created belong to the project that the credentials are linked to.
+
+
+## Object identity in {{brand}}
+
+Regardless of whether you access a specific object via the Swift API or the S3 API, *it is the same object.*
+This also includes that the object's *container* (accessed via the Swift API) is identical to its containing *bucket* (accessed via the S3 API).
+
+Thus, if you
+
+* [enable the OpenStack CLI](../howto/getting-started/enable-openstack-cli.md) with a certain set of user credentials;
+* use those credentials to [create a private container](../howto/object-storage/swift/private-container.md) named `test` and an object named `my‑object`, using the Swift API;
+* use the same OpenStack credentials to [create AWS-style credentials](../howto/object-storage/s3/credentials.md);
+* then use those AWS-style credentials to list your buckets and objects, using the S3 API,
+
+you will find an S3 bucket named `test`, containing an object named `my‑object`.
+
+
+## Permission and feature conflicts
+
+The fact that objects are identical when accessed via Swift and S3 also entails that bucket and object permissions, set via one API, also apply to object access using the other API.
```

Comment:
```suggestion
The fact that objects are identical when accessed via Swift and S3, entails that bucket and object permissions, set via one API, also apply to object access using the other API.
```

Reply:
No, I think "also" is better than the comma here.